configuration.rst

```diff
@@ -131,6 +133,20 @@ db-extra-search-path
 
   Multiple schemas can be added in a comma-separated string, e.g. ``public, extensions``.
 
+.. _db-channel:
+
+db-channel
+----------
+
+  The name of the notification channel that PostgREST uses to re read the config file and reload the schema cache. For more information, see :ref:`schema_reloading`.
+
+.. _db-channel-enabled:
+
+db-channel-enabled
+------------------
+
+  When this is set to :code:`true`, the notification channel specified in :ref:`db-channel` is enabled.
```

Comment:
We should mention here that this setting is not compatible with connection poolers using the transaction mode.